glotaran/parameter/test/test_parameter.py

```diff
@@ -488,97 +346,26 @@ def test_parameter_numpy_operations():
     assert parm1 <= parm2
 
 
-TEST_CSV = """
-label, value, minimum, maximum, vary, non-negative, expression
-rates.k1,0.050,0,5,True,True,None
-rates.k2,None,-inf,inf,True,True,$rates.k1 * 2
-rates.k3,2.311,-inf,inf,True,True,None
-pen.eq.1,1.000,-inf,inf,False,False,None
-"""
-
-
-def test_param_group_from_csv(tmpdir):
-
-    csv_path = tmpdir.join("parameters.csv")
-    with open(csv_path, "w") as f:
-        f.write(TEST_CSV)
-
-    params = load_parameters(csv_path)
-
-    assert "rates" in params
-
-    assert params.has("rates.k1")
-    p = params.get("rates.k1")
-    assert p.label == "k1"
-    assert p.value == 0.05
-    assert p.minimum == 0
-    assert p.maximum == 5
-    assert p.vary
-    assert p.non_negative
-    assert p.expression is None
-
-    assert params.has("rates.k2")
-    p = params.get("rates.k2")
-    assert p.label == "k2"
-    assert p.value == params.get("rates.k1") * 2
-    assert p.minimum == -np.inf
-    assert p.maximum == np.inf
-    assert not p.vary
-    assert not p.non_negative
-    assert p.expression == "$rates.k1 * 2"
-
-    assert params.has("rates.k3")
-    p = params.get("rates.k3")
-    assert p.label == "k3"
-    assert p.value == 2.311
-    assert p.minimum == -np.inf
-    assert p.maximum == np.inf
-    assert p.vary
-    assert p.non_negative
-    assert p.expression is None
-
-    assert "pen" in params
-    assert "eq" in params["pen"]
-
-    assert params.has("pen.eq.1")
-    p = params.get("pen.eq.1")
-    assert p.label == "1"
-    assert p.value == 1.0
-    assert p.minimum == -np.inf
-    assert p.maximum == np.inf
-    assert not p.vary
-    assert not p.non_negative
-    assert p.expression is None
-
-
-def test_parameter_to_csv(tmpdir):
-    csv_path = tmpdir.join("parameters.csv")
-    params = load_parameters(
-        """
-    b:
-        - ["1", 0.25, {vary: false, min: 0, max: 8}]
-        - ["2", 0.75, {expr: '1 - $b.1', non-negative: true}]
-    rates:
-        - ["total", 2]
-        - ["branch1", {expr: '$rates.total * $b.1'}]
-        - ["branch2", {expr: '$rates.total * $b.2'}]
-    """,
-        format_name="yml_str",
+def test_parameter_to_from_dict():
```

Comment:
```suggestion
def test_parameter_dict_roundtrip():
```
This might be a more meaningful name.